templates/container.cfg.xml.erb

```diff
@@ -30,6 +33,7 @@
 <%- if @jmx_reset_time -%>
         jmx-reset-time="<%= @jmx_reset_time -%>"
 <%- end -%>
+        repose-version="<%= @repose_version %>"
```

Comment:
This would be conditionally added to the element mentioned above.

Reply:
Addressed this in amended commit